projects/Mallard/src/screens/home-screen.tsx

```diff
@@ -86,7 +87,7 @@ const IssueList = withNavigation(
                 <>
                     <BaseList
                         style={{ paddingTop: 0 }}
-                        data={issueList}
+                        data={[...issueList, ...issueList]}
```

Comment:
Double issueList?

Reply:
Oh! Thanks, this was for testing - thanks!